client/src/components/formHelpers/form.tsx

```diff
@@ -69,27 +68,25 @@ function formatUrlValues(
   return validatedValues;
 }
 
-export type FormProps = {
+export type StrictSolutionFormProps = {
   buttonText?: string;
```

Comment:
this type isn't true, but I am planning to delete that file in future PR, because there is another file that is doing the same thing in another file.